### PR TITLE
Adding null check for Activity

### PR DESCRIPTION
**Pull request by nedarokt**

### CODE REVIEW

**Comment by danial-rokt** [comment]
@nedarokt Do you think it is a good idea to check if currentActivity is not null on the execute method too?
Just to avoid any unknown crashes

**Comment by nedarokt** [comment]
@danial-rokt we don't use it in execute

File: Rokt.Widget/android/src/main/java/com/reactlibrary/RNRoktWidgetModule.java [hunk 1]
```diff
@@ -44,27 +44,34 @@
     @ReactMethod
     public void initialize(String roktTagId, String appVersion) {
         Activity currentActivity = getCurrentActivity();
-        Rokt.INSTANCE.init(roktTagId, appVersion, currentActivity);
+        if (currentActivity != null && roktTagId != null) {
```
Comment (danial-rokt) [inline Rokt.Widget/android/src/main/java/com/reactlibrary/RNRoktWidgetModule.java:47]: A silly question, if roktTagId could be null, would it be same for appVersion?

Comment (nedarokt) [reply Rokt.Widget/android/src/main/java/com/reactlibrary/RNRoktWidgetModule.java:47]: not silly at all, I just missed that! Will update now

**Review by danial-rokt — APPROVED** [review]

**Pull request merged**